Not working:
```
__________________________ test_exec_interactive_env ___________________________

    @pytest.mark.skipif(sys.platform == 'win32',
                        reason="'dcos task exec' not supported on Windows")
    def test_exec_interactive_env():
        task_id = _get_task_id('test-app1')
    
        command = ['dcos', 'task', 'exec', '--interactive', task_id, 'env']
        returncode, stdout, _ = exec_command(command)
        assert returncode == 0
>       assert b'TERM' in stdout
E       AssertionError: assert b'TERM' in b'LIBPROCESS_SSL_REQUIRE_CERT=false\nLIBPROCESS_SSL_VERIFICATION_DEPTH=4\nLIBPROCESS_SSL_ECDH_CURVES=auto\nMESOS_FRAME...=/test-app1\nLIBPROCESS_SSL_ENABLE_TLS_V1_1=false\nLIBPROCESS_SSL_ENABLE_TLS_V1_0=false\nLIBPROCESS_SSL_ENABLED=true\n'

tests/integrations/test_task.py:335: AssertionError
----------------------------- Captured stdout call -----------------------------
CMD: ['dcos', 'task', '--json', 'test-app1']
STDOUT: [
  {
    "discovery": {
      "name": "test-app1",
      "ports": {
        "ports": [
          {
            "name": "default",
            "number": 24117,
            "protocol": "tcp"
          }
        ]
      },
      "visibility": "FRAMEWORK"
    },
    "executor_id": "",
    "framework_id": "cc54a862-534b-4d3d-a5c0-654165d27b3d-0001",
    "id": "test-app1.42351ddf-459d-11e9-bc3e-d64726fe47a1",
    "labels": [
      {
        "key": "PACKAGE_ID",
        "value": "test-app"
      },
      {
        "key": "PACKAGE_VERSION",
        "value": "1.2.3"
      },
      {
        "key": "DCOS_SPACE",
        "value": "/test-app1"
      }
    ],
    "name": "test-app1",
    "resources": {
      "cpus": 0.1,
      "disk": 0.0,
      "gpus": 0.0,
      "mem": 16.0,
      "ports": "[24117-24117]"
    },
    "role": "slave_public",
    "slave_id": "cc54a862-534b-4d3d-a5c0-654165d27b3d-S0",
    "state": "TASK_RUNNING",
    "statuses": [
      {
        "container_status": {
          "container_id": {
            "value": "6dd38ed6-b0c1-4219-b34c-9a36b351560c"
          },
          "network_infos": [
            {
              "ip_addresses": [
                {
                  "ip_address": "10.10.0.163",
                  "protocol": "IPv4"
                }
              ]
            }
          ]
        },
        "state": "TASK_STARTING",
        "timestamp": 1552487734.109557
      },
      {
        "container_status": {
          "container_id": {
            "value": "6dd38ed6-b0c1-4219-b34c-9a36b351560c"
          },
          "network_infos": [
            {
              "ip_addresses": [
                {
                  "ip_address": "10.10.0.163",
                  "protocol": "IPv4"
                }
              ]
            }
          ]
        },
        "state": "TASK_RUNNING",
        "timestamp": 1552487734.11341
      }
    ]
  }
]

STDERR: 
CMD: ['dcos', 'task', 'exec', '--interactive', 'test-app1.42351ddf-459d-11e9-bc3e-d64726fe47a1', 'env']
STDOUT: LIBPROCESS_SSL_REQUIRE_CERT=false
```

--- a/python/lib/dcos/dcos/mesos.py
+++ b/python/lib/dcos/dcos/mesos.py
@@ -1451,6 +1451,14 @@ class TaskIO(object):
                     'container'] = {
                         'type': 'MESOS',
                         'tty_info': {}}
+            message[
+                'launch_nested_container_session'][
+                    'command'][
+                        'environment'] = {
+                            'variables': [{
+                                    'name': 'TERM',
+                                    'type': 'VALUE',
+                                    'value': 'xterm'}]}
 
         req_extra_args = {
             'stream': True,

